DeviceAdapters/TeledyneDalsaGigE/TestCamera.h

```diff
@@ -0,0 +1,154 @@
+///////////////////////////////////////////////////////////////////////////////
+// FILE:          TestCamera.h
+// PROJECT:       Micro-Manager
+// SUBSYSTEM:     DeviceAdapters
+//-----------------------------------------------------------------------------
+// DESCRIPTION:   Skeleton code for the micro-manager camera adapter. Use it as
+//                starting point for writing custom device adapters
+//                
+// AUTHOR:        Nenad Amodaj, http://nenad.amodaj.com
+//                
+// COPYRIGHT:     University of California, San Francisco, 2011
+//
+// LICENSE:       This file is distributed under the BSD license.
+//                License text is included with the source distribution.
+//
+//                This file is distributed in the hope that it will be useful,
+//                but WITHOUT ANY WARRANTY; without even the implied warranty
+//                of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+//
+//                IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+//                CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+//                INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES.
+//
+
+#ifndef _TestCamera_H_
```

Comment:
all `TestCamera` instances should be replaced by a more specific descriptor (including file names)

Reply:
resolved.